lib/mapping.json

```diff
@@ -7,7 +7,6 @@
       },
     
     "compilers" : {
-        "0.5.7" :   "v0.5.7+commit.6da8b019",
```

Comment:
I assume you remove this for testing purpose. I think we can keep this mapping as it is and introduce a sample mapping file dedicated for tests.

Reply:
yes, I removed it for the testing purpose.

How we'll manage to use the sample mapping file for tests and original mapping file for the package?

Reply:
We can change our approach a bit. We can import `mapping` in `verify.js` and pass it as a function parameter to `solReleases`. Further we can add the compiler search related tests separately for `solReleases.js`. Here we can pass mock mapping in the tests.

Reply:
sounds good